registry.json

```diff
@@ -3898,5 +3898,19 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+  {
+    "ticker": "DEEP",
+    "poolId": "pool136903m0kag2amlga2ypcz8wnf7d552qgy5t6vz2pqlzuq2kyshy",
```

Comment:
Please change this to the pool_id, e.g. 8e8af8edf6ea15ddfd1d5103811dd34f9b4a28082517a6094107c5c0